docs/LOCALHOST.md

```diff
@@ -19,15 +19,10 @@ Instructions for running a local "developer" environment on localhost:
    * [PostgreSQL](../Meadowlark-js/backends/meadowlark-postgresql-backend/docker)
    * [OpenSearch](../Meadowlark-js/backends/meadowlark-opensearch-backend/docker)
 6. Setup environment variables for running
-   [meadowlark-fastify](../Meadowlark-js/services/meadowlark-fastify/) service.
-   The folder has an `example.env` file with all settings needed to run the
-   service; the easiest way to set your environment variables is to duplicate
-   this file in the folder and rename the file to `.env`. Review the settings in
-   the `.env` file to see what values, if any, to change. In particular:
-      * `DOCUMENT_STORE_PLUGIN` - Make sure your chosen backend is uncommented
-        and comment out any others.
-      * `QUERY_HANDLER_PLUGIN` and `LISTENER1_PLUGIN` - Uncomment these two for
-        GET query support using OpenSearch.
+   [meadowlark-fastify](../Meadowlark-js/services/meadowlark-fastify/readme.md) service.
+7. Open a command prompt and navigate to the `/Meadowlark-js/services/meadowlark-fastify` folder
+8. Run `yarn start:local` to start the fastify service
```

Comment:
Don't they need to `yarn build` first?

Reply:
yes, for some reason I though that was already in the instructions someplace. I added steps to build in the localhost instructions and then added them in the fastify instructions with a note that they can skip if they've already built